source/update.rst

```diff
@@ -5,97 +5,105 @@ Upgrade Percona XtraDB Cluster
 
 Starting from version 1.1.0, Percona Kubernetes Operator for Percona
 XtraDB Cluster allows upgrades to newer versions. This includes upgrades of the
-Operator itself, and upgrades of the Percona XtraDB Cluster.
+Percona XtraDB Cluster, and upgrades of the Operator itself.
 
 .. contents:: :local:
 
-.. _operator-update:
```

Comment:
right order is:
1. update of operator (without manual update of pxc cluster)
2. smart update of pxc cluster 
3. outdated manual update of pxc cluster without operator update